components/link/link-mixin.js

```diff
@@ -0,0 +1,60 @@
+export const LinkMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * When present, this attribute instructs browsers to download a URL
+			 * instead of navigating to it, so the user will be prompted to save
+			 * it as a local file.
+			 */
+			download: {
+				type: Boolean,
+				reflectToAttribute: true
+			},
+			/**
+			 * Contains a URL or a URL fragment that the hyperlink points to.
+			 * Corresponds to the `href` attribute on standard `<a>` elements.
+			 */
+			href: {
+				type: String,
+				reflectToAttribute: true
+			},
```

Comment:
Are all of these necessary, or could we get by with just `href` for now? Also, I think `reflectToAttribute` isn't needed anywhere, unless you're using CSS attribute selectors somewhere.

Reply:
- [ ] reflectToAttribute change to reflect
- [ ] Double check if I am using the href here.
- [ ] Make sure this is converted better to lit.

I probably am using a css selector for it since the style changes. However, I even if I am I don't think I need this.